lib/src/types/easings.dart

```diff
@@ -1,13 +1,28 @@
+import 'dart:math';
+
 class StrokeEasings {
   /// Identity function.
-  /// Returns the input value.
+  /// Previously used as the default ease in [getStrokeRadius].
+  @Deprecated('Use easeMiddle or define your own easing function')
   static double identity(double t) => t;
 
-  /// Ease-in-out function.
-  /// Used for the taper start.
+  /// This function is actually an ease-out function.
+  /// Previously used for the taper start.
+  @Deprecated('Use easeMiddle or define your own easing function')
   static double easeInOut(double t) => t * (2 - t);
 
-  /// Ease-out-cubic function.
-  /// Used for the taper end.
+  /// This function does not output a value between 0 and 1.
+  /// Previously used for the taper end.
+  @Deprecated('Use easeMiddle or define your own easing function')
   static double easeOutCubic(double t) => --t * t * t + 1;
+
+  /// Easing function that slows down around the middle.
+  static double easeMiddle(double t) {
+    // Take average of t and cubic ease of t to smooth out the curve.
+    return (t + _easeMiddleCubic(t)) / 2;
+  }
+
+  static double _easeMiddleCubic(double t) {
+    return 4 * pow(t - 0.5, 3) + 0.5;
+  }
```

Comment:
Is there a reason why we wouldn't easeMiddle as an additional easing here, rather than deprecate the others?

Reply:
I thought it would be better for devs to define the functions in their own code, rather than in this project where they might be removed in the future since we're not using them